<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control</class>
 <widget class="QDialog" name="Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>370</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>100</number>
    </property>
    <property name="leftMargin">
     <number>150</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>150</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>60</y>
     <width>581</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="1" column="3">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>目的城市：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>起飞时间：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLineEdit" name="ToAirport"/>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>降落机场：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="swcpiaojia"/>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商务舱票数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="LEbianhao"/>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="jjcpiaoshu"/>
    </item>
    <item row="1" column="4" colspan="2">
     <widget class="QComboBox" name="combTO"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>经济舱票数：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>经济舱票价：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="jjcpiaojia"/>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QLineEdit" name="LEticketcount"/>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="swcpiaoshu"/>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>到达时间：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>航班编号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="combGO"/>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商务舱票价：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>总 票 数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>出发城市：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit"/>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="GoAirport"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>起飞机场：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QDateTimeEdit" name="dateTimeEdit_2">
      <property name="date">
       <date>
        <year>2000</year>
        <month>2</month>
        <day>12</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>增加航班</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>修改航班</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>删除航班</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>查询航班</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>42</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>LEbianhao</tabstop>
  <tabstop>LEticketcount</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>dateTimeEdit_2</tabstop>
  <tabstop>jjcpiaoshu</tabstop>
  <tabstop>swcpiaoshu</tabstop>
  <tabstop>jjcpiaojia</tabstop>
  <tabstop>swcpiaojia</tabstop>
  <tabstop>GoAirport</tabstop>
  <tabstop>ToAirport</tabstop>
  <tabstop>combGO</tabstop>
  <tabstop>combTO</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
